### PR TITLE
Translation update v8.3.9.0 (ru-ru)

### PR DESCRIPTION
**Pull request by iDolmatov**

### CODE REVIEW

File: src/OfficeToolPlus/Style/Languages/ru-ru.xaml [hunk 1]
```diff
@@ -31,7 +31,7 @@
     <sys:String x:Key="OK">OK</sys:String>
     <sys:String x:Key="Cancel">Отмена</sys:String>
     <!-- TO DO: Translate -->
```
Comment (YerongAI) [inline src/OfficeToolPlus/Style/Languages/ru-ru.xaml:33]: "TO DO" remark doesn't removed after done.

File: src/OfficeToolPlus/Style/Languages/ru-ru.xaml [hunk 2]
```diff
@@ -105,7 +105,7 @@
     <sys:String x:Key="DeployProductID">ID продукта:</sys:String>
     <sys:String x:Key="DeployAddProduct">Добавить продукт</sys:String>
     <!-- TO DO: Translate -->
```
Comment (YerongAI) [inline src/OfficeToolPlus/Style/Languages/ru-ru.xaml:107]: "TO DO" remark doesn't removed after done.

File: src/OfficeToolPlus/Style/Languages/ru-ru.xaml [hunk 3]
```diff
@@ -130,7 +130,7 @@
     <sys:String x:Key="DeployAsProof">Развернуть только модули проверки.</sys:String>
     <sys:String x:Key="DeployAddLanguage">Добавить язык</sys:String>
     <!-- TO DO: Translate -->
```
Comment (YerongAI) [inline src/OfficeToolPlus/Style/Languages/ru-ru.xaml:132]: "TO DO" remark doesn't removed after done.

**Review by YerongAI — CHANGES_REQUESTED** [review]
Line 348, the TO DO task doesn't finish yet, please check again.

**Comment by iDolmatov** [comment]
Fixed the TO DO tasks (4).

**Pull request merged**